src/DynamicRupture/Output/ReceiverBasedOutput.cpp

```diff
@@ -433,25 +449,26 @@ real ReceiverOutput::computeRuptureVelocity(Eigen::Matrix<real, 2, 2>& jacobiT2d
 
     auto* rt = getCellData(local, drDescr->ruptureTime);
     for (size_t jBndGP = 0; jBndGP < misc::NumBoundaryGaussPoints; ++jBndGP) {
-      const real chi = chiTau2dPoints(jBndGP, 0);
-      const real tau = chiTau2dPoints(jBndGP, 1);
+      const real chi = seissol::multisim::multisimTranspose(chiTau2dPoints, 0, jBndGP);
+      const real tau = seissol::multisim::multisimTranspose(chiTau2dPoints, 1, jBndGP);
+
       basisFunction::tri_dubiner::evaluatePolynomials(phiAtPoint.data(), chi, tau, NumPoly);
 
       for (size_t d = 0; d < NumDegFr2d; ++d) {
         projectedRT[d] +=
             seissol::multisim::multisimWrap(weights, 0, jBndGP) * rt[jBndGP] * phiAtPoint[d];
       }
     }
-
     auto m2inv =
         seissol::init::M2inv::view::create(const_cast<real*>(seissol::init::M2inv::Values));
     for (size_t d = 0; d < NumDegFr2d; ++d) {
       projectedRT[d] *= m2inv(d, d);
     }
 
-    const real chi = chiTau2dPoints(local.nearestInternalGpIndex, 0);
-    const real tau = chiTau2dPoints(local.nearestInternalGpIndex, 1);
-
+    const real chi =
```

Comment:
Cf. the comment above; the `multisimTranspose` transposes for fused, and does _not_ transpose for serial.